content/docs/command-reference/exp/show.md

```diff
@@ -53,6 +53,20 @@ sorted inside each group, chronologically by default. The `--sort-by` and
 `--sort-order` options can change this ordering, based on any single, visible
 metric or param.
 
+### Parallel Coordinates Plot
+
+When the `--pcp` option is passed, an interactive Parallel Coordinates Plot will
+be generated using the data from the table.
+
+This plot is useful to explore the relationships between the metrics _and_
+params used in experiments.
+
+![](/img/user_guide_parallel_coordinates.gif) _Parallel Coordinates Plot_
```

Comment:
> Unfortunately, it's pretty hard to read the example images because they are so small. Not sure if there's anything we can do about it 🤔 .

@dberenbaum Do you refer to this GIF? 

For the other images, I am able to click and open a full page view (where I am able to read all letters)

Reply:
Yeah, I can open them, but they look small when embedded within the page.

Reply:
Hacked the plotly layout a bit to increase height and font size (also updated plots to use same examples as the above tables, from example-get-started).

Line width, unfortunately, is a need sponsor feature https://github.com/plotly/plotly.js/issues/2573 😅

Reply:
Font size was the real concern. Looks better now, thanks!

Reply:
> > hard to read the example images because they are so small
>
> Do you refer to this GIF?

I do think the GIF could use a smaller example. Maybe just 2 params and a single metric ([example](https://docs.wandb.ai/ref/app/features/panels/parallel-coordinates)). I'd also reduce the number of experiments (7 max) if possible so this sample is short.

The ones in the longer example seem fine to me since you can click on them to examine them (Still preferably a few less experiments would be ideal IMO, but up to you).

Reply:
I am not sure about reducing the number of experiments. 

I think that the main usage is when you have many experiments and the plot helps to perceive some patterns more easily than the table.

Reply:
Agree on reducing to 2 params 1 metric

Reply:
I will wait for https://github.com/iterative/dvc/pull/7141 in order to update the plot (to perhaps remove the Experiment column). Leaving the current GIF for now, it that's ok